apps/src/templates/Notification.jsx

```diff
@@ -19,10 +19,12 @@ const styles = {
   main: {
     borderWidth: 1,
     borderStyle: 'solid',
-    height: 72,
+    minHeight: 72,
```

Comment:
Use min height so that if we have more than one line of text, the box will grow.